base/fastmath.jl

```diff
@@ -124,6 +125,38 @@ function make_fastmath(symb::Symbol)
 end
 make_fastmath(expr) = expr
 
+"""
+@fastmath expr
```

Comment:
Actually, this should be a code block (indent it) and this should be added to the doc/stdlib/